### PR TITLE
CORE-1143: add `is_favorite` to app listings

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -92,7 +93,7 @@ app.get("/users/:username", async (req, res) => {
         const retval = {
             apps: {
                 recentlyAdded: await recentlyAddedData(db, username, limit),
-                public: await publicAppsData(db, limit),
+                protected: await publicAppsData(db, username, limit),
```
Comment (ianmcorvidae) [inline src/index.js]: I'm confused about why this says `protected` now, not sure if that's an intended change.

**Review by ianmcorvidae — APPROVED** [review]
One comment that might be an unintended change, the rest seems good. I'm trusting that you know how favorites are stored in the DB, because I admit I haven't researched that part.

File: src/index.js (hunk 1 above)
Comment (slr71) [reply src/index.js]: That was unintended. I'm not sure why that changed either. 😕 Thanks for catching that.

**Comment by slr71** [comment]
Thanks for the review! Merging this now.

**Pull request merged**